Scripts/Core/EditorVR.Interfaces.cs

```diff
@@ -79,6 +86,10 @@ internal void ConnectInterfaces(object obj, Transform rayOrigin = null)
 					if (ray != null)
 						ray.rayOrigin = rayOrigin;
 
+					var handedRay = obj as IUsesHandedRayOrigin;
```

Comment:
Let's separate these two interfaces, since we already have IUsesRayOrigin. Might as well have IUsesNode.